components/operator/cmd/main.go

```diff
@@ -74,6 +75,9 @@ func main() {
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&region, "region", "eu-west-1", "The cloud region in use for the operator")
 	flag.StringVar(&env, "env", "staging", "The current environment in use for the operator")
+	flag.StringVar(&licence.Token, "licence-token", "", "The licence token")
+	flag.StringVar(&licence.Issuer, "licence-issuer", "", "The licence issuer")
+	flag.StringVar(&licence.ClusterID, "licence-cluster-id", "", "The licence cluster ID")
```

Comment:
You should get this information dynamically. Otherwise it is easy for users to configure what they want and reuse the same id across many clusters.
As the kube-system namespace is created at the beginning of a cluster, it is really complicated to change.